src/proxy/proxy.cpp

```diff
@@ -565,3 +574,14 @@ bool Proxy::isGmcpModuleEnabled(const GmcpModuleTypeEnum &module) const
 {
     return m_userTelnet->isGmcpModuleEnabled(module);
 }
+
+void Proxy::slot_onSendGameTimeToClock(const int year,
+                                       const std::string &monthStr,
+                                       const int day,
+                                       const int hour)
+{
+    const int month = m_mumeClock.getMumeMonth(::toQStringLatin1(monthStr));
+    MumeMoment moment(year, month, day, hour, 0);
+    QString mumeTime = m_mumeClock.toMumeTime(moment);
+    m_mumeClock.parseMumeTime(mumeTime);
```

Comment:
Doesn't this adjust the granularity of the clock? We're not aware of the seconds marker so we should keep it fuzzy.

Reply:
I didn't think about that, I'll change it.

Reply:
So 
`m_mumeClock.setPrecision(MumeClockPrecisionEnum::DAY);`
?

Reply:
Well, you're basically taking a MumeTime and then changing it to a String and then parsing that String. I would avoid the string transformation entirely and just build a new function all to do what you want.

MumeClock::parseMSSP(year, month, day, hour)